src/styled/globalStyle.tsx

```diff
@@ -0,0 +1,17 @@
+import  { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
Will we have (outside this function) exported global variables  like: HEADER_FONT, CONTENT_FONT ready to use in the project?